recipes/ucsc-bedclip/build.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
-mkdir -p "$PREFIX/bin"
-export MACHTYPE=x86_64
+
+if [ $(arch) = "aarch64" ]
+then
+    export CFLAGS+=" -O3 -mcpu=neoverse-n1 "
```

Comment:
Why `neoverse-n1`? Isn't it too specific ?

Reply:
Hey, sorry about the too-specific default. We have a Graviton instance where we test these recipes and that value seemed reasonable at that point, but I agree it might be constraining.

Do you have any idea for a more generic but still valid defaults?. I've found [this](https://gcc.gnu.org/onlinedocs/gcc-11.3.0/gcc/AArch64-Options.html) link with possible values, but don't know which ones to choose at this point. Maybe a valid approach might be to let another reviewer to propsose different flags when they feel the need of doing so?.

Reply:
IMO `-march=`, `-mtune=` and `-mcpu=` should not be set at all.
As the same page says: `Where none of -mtune=, -mcpu= or -march= are specified, the code is tuned to perform well across a range of target processors. `

Reply:
Done [here](https://github.com/bioconda/bioconda-recipes/pull/46686/commits/255ab096784338a38427981e877894e4e1048b6f). Please tell me if I need to rebase the branch or anything.

Reply:
Rebasing would be good for sure!
Currently https://github.com/bioconda/bioconda-recipes/pull/46686/files shows many changes which were already merged.
Rebasing to latest `master` will show a diff only  for this particular change.

Reply:
rebased [here](https://github.com/bioconda/bioconda-recipes/pull/46686) (I think I cannot use this PR since it's already closed)